src/test/run-make-fulldeps/rustdoc-map-file/expected.json

```diff
@@ -0,0 +1,5 @@
+{
+  "foo/macro.foo!.html": "foo/macro.foo.html",
+  "foo/struct.Quz.html": "foo/struct.Quz.html",
+  "foo/struct.Bar.html": "foo/struct.Bar.html"
```

Comment:
This doesn't look right, these redirect to the same file.

Reply:
Not exactly: normally those types are wrapped in submodules. The point of this test is to check that the generated path is the correct one.

Reply:
I realized my mistake: I'll fix it. It should be the full path including the private item!